src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
+
+# Installing Ceph
+
+Here we present one way to install a rudimentary Ceph cluster with Juju backed
+by an AWS cloud. See [Ceph documentation][upstream-ceph-documentation] to find
+out more about Ceph.
```

Comment:
No, but I'll assume he gets your mention and replies if he has anything to add. To be clear, I didn't intend this to become a sophisticated "how to install Ceph with Juju". If it's warranted we could have such a thing elsewhere in the Juju docs (or as a tutorial?).